src/main/kotlin/com/terraformation/backend/admin/AdminEmailController.kt

```diff
@@ -0,0 +1,70 @@
+package com.terraformation.backend.admin
+
+import com.terraformation.backend.api.RequireGlobalRole
+import com.terraformation.backend.config.TerrawareServerConfig
+import com.terraformation.backend.customer.db.UserStore
+import com.terraformation.backend.customer.model.IndividualUser
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.email.EmailService
+import com.terraformation.backend.email.model.DocumentsUpdate
+import com.terraformation.backend.log.perClassLogger
+import jakarta.validation.constraints.NotBlank
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.servlet.mvc.support.RedirectAttributes
+
+@Controller
+@RequestMapping("/admin")
+@RequireGlobalRole([GlobalRole.SuperAdmin])
+@Validated
+class AdminEmailController(
+    private val config: TerrawareServerConfig,
+    private val emailService: EmailService,
+    private val userStore: UserStore,
+) {
+  private val log = perClassLogger()
+
+  @GetMapping("/email")
+  fun getSendTestEmailPage(model: Model): String {
+    model.addAttribute("emailNames", listOf("DocumentsUpdate"))
+
+    return "/admin/email"
+  }
+
+  @PostMapping("/sendEmail")
+  fun sendTestEmail(
+      @NotBlank @RequestParam emailName: String,
+      @NotBlank @RequestParam recipient: String,
+      redirectAttributes: RedirectAttributes,
+  ): String {
+    val user = userStore.fetchByEmail(recipient) as IndividualUser
+
+    try {
+      emailService.sendUserNotification(
+          user,
+          when (emailName) {
+            "DocumentsUpdate" -> DocumentsUpdate(config)
+            else -> throw IllegalArgumentException("Invalid test email name $emailName")
+          },
+          false,
+      )
+
+      redirectAttributes.successMessage =
+          if (config.email.enabled) {
```

Comment:
Might also be nice to check this on the GET endpoint and include a message to warn people. Not that email will be disabled in staging or production, of course.